BAZProjectC1/ModalController/FiltersMovieModalViewController.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  FiltersMovieModalViewController.swift
+//  BAZProjectC1
+//
+//  Created by 961184 on 22/09/22.
+//
+
+import UIKit
+import Foundation
+
+class FiltersMovieModalViewController: UIViewController{
+
+    @IBOutlet weak var btnCloseModal: UIButton!
+    @IBOutlet weak var txtFilterCategory: UITextField!
+    @IBOutlet weak var txtLanguage: UITextField!
+    @IBOutlet weak var btnAgree: UIButton!
+    @IBOutlet weak var btnCancel: UIButton!
+    
+    private var strPrincipalAction: String?
+    private var strSecundaryAction: String?
+    private var bHideSecundaryButton: Bool = false
+    private var delegateSelectedFilter: SelectedFilterProtocol?
+    
+    private let filterCategoryPicker = UIPickerView()
+    private let filterLanguagePicker = UIPickerView()
+    private let categoryPickerData: [CategoryMovieType] = [.trending, .nowPlaying, .popular, .topRated, .upcoming]
+    private let languagePickerData: [ApiLanguageResponse] = [.es, .de, .en, .pt]
+    
+    private var selectedTextFiel: UITextField?
+    private var selectedCategoryPicker: CategoryMovieType?
+    private var selectedLanguagePicker: ApiLanguageResponse?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.btnAgree.setTitle(strPrincipalAction, for: .normal)
+        self.btnCancel.setTitle(strSecundaryAction, for: .normal)
+        self.setupUIPickers()
+        if bHideSecundaryButton{
+            self.btnCancel.isHidden = bHideSecundaryButton
+        }
```

Comment:
`self.btnCancel.setTitle(strSecundaryAction, for: .normal)` podria ir en el else de `bHideSecundaryButton`
para evitar la accion, si va a estar oculto de todas formas